lib/beaker/dsl/helpers/puppet_helpers.rb

```diff
@@ -714,35 +716,51 @@ def wait_for_host_in_dashboard(host)
 
         # Ensure the host has requested a cert, then sign it
         #
-        # @param [Host, Array<Host>, String, Symbol] host    One or more hosts to act upon,
+        # @param [Host, Array<Host>, String, Symbol] host   One or more hosts to validate signing certificate,
         #                            or a role (String or Symbol) that identifies one or more hosts.
+        #                            No argument, or an empty array means no validation of success
+        #                            for specific hosts will be performed.
         #
         # @return nil
         # @raise [FailTest] if process times out
-        def sign_certificate_for(host)
-          block_on host do | host |
-            if [master, dashboard, database].include? host
+        def sign_certificate_for(host = [])
```

Comment:
The change on this line stops someone getting an arity exception if they pass nothing for "host". I suspect the method would now just do nothing at all if someone made that mistake? I'm not sure if this is worth worrying about though - the method name plus the yardoc makes it sound strange that anyone would expect to not pass a value for host